src/client/game.cpp

```diff
@@ -4217,6 +4193,35 @@ void Game::updateFrame(ProfilerGraph *graph, RunStats *stats, f32 dtime,
 	g_profiler->avg("Game::updateFrame(): update frame [ms]", tt_update.stop(true));
 }
 
+void Game::updateClouds(float dtime)
+{
+	if (this->sky->getCloudsVisible()) {
+		this->clouds->setVisible(true);
+		this->clouds->step(dtime);
+		// camera->getPosition is not enough for 3rd person views
+		v3f camera_node_position = this->camera->getCameraNode()->getPosition();
+		v3s16 camera_offset      = this->camera->getOffset();
+		camera_node_position.X   = camera_node_position.X + camera_offset.X * BS;
+		camera_node_position.Y   = camera_node_position.Y + camera_offset.Y * BS;
+		camera_node_position.Z   = camera_node_position.Z + camera_offset.Z * BS;
+		this->clouds->update(camera_node_position,
+				this->sky->getCloudColor());
```

Comment:
This line shouldn't be wrapped. (The limit is 95 characters according to https://dev.minetest.net/Code_style_guidelines#Don.27t_let_things_get_too_large.)